src/content/developers/docs/oracles/index.md

```diff
@@ -4,61 +4,61 @@ description: Oracles provide Ethereum smart contracts with access to real-world
 lang: en
 ---
 
-Oracles are data feeds that bring data from off the blockchain (off-chain) data sources and puts it on the blockchain (on-chain) for smart contracts to use. This is necessary because smart contracts running on Ethereum cannot access information stored outside the blockchain network.
+Oracles are data feeds that make data from off-chain sources available to the blockchain for smart contracts to use. This is necessary because Ethereum-based smart contracts cannot, by default, access information stored outside the blockchain network.
```

Comment:
Agree this is better, but the wording tripped me up on first read. What about this @setgree:

```suggestion
Oracles are data feeds that make off-chain data sources available to the blockchain for smart contracts. This is necessary because Ethereum-based smart contracts cannot, by default, access information stored outside the blockchain network.
```

Reply:
Q: are smart-contracts Ethereum-based or do they run atop Ethereum? (and is there a difference?)

Bikeshedding mostly 😁

Reply:
anything technical like this, I completely to defer to you; I have no idea if the distinction is meaningful